service-book-api/components/queries/adhocQueries.js

```diff
@@ -0,0 +1,17 @@
+const { join } = require('path');
+const booksReducers = require('require-all')(join(__dirname, '..', '..', 'models', 'books', 'projection', 'reducers'));
+
+module.exports = store => {
```

Comment:
This is a projection, we should treat it as such. I will merge this and make the suggestion